plugin/files/airconsole/edittime.js

```diff
@@ -37,11 +37,11 @@ AddCmpParam("Is", "Compare message data key");
 AddAnyTypeParam("Value", "Compare to this value");
 AddCondition(0, cf_trigger, "On message key", "Data", "On message key is {2}", "Triggered when a message key is received from a device.", "OnMessageKey");
 
-// Receive onMessage
+// Receive onMessageIs
 AddAnyTypeParam("Value", "Compare to this value");
 // AND DEVICE ID IS
 AddAnyTypeParam("Object DeviceID", "The object associated to this device id");
-AddCondition(1, cf_trigger, "On message", "Data", "On message is {0} and from {1}", "Triggered when a message is received from a device.", "OnMessage");
+AddCondition(1, cf_trigger, "On specific message", "Data", "On message is {0} and from {1}", "Triggered when a specific message is received from a device.", "OnMessageIs");
```

Comment:
Yes it is. Reverted back to master, created anew  project, just added OnMessage, saved, closed, reinjected the updated one, reopened and it's ok